x-pack/plugins/maps/public/shared/layers/tooltips/es_tooltip_property.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { buildPhraseFilter } from '@kbn/es-query';
+import { filterBarQueryFilter } from '../../../kibana_services';
+import { TooltipProperty } from './tooltip_property';
+
+export class ESTooltipProperty extends TooltipProperty {
+
+  constructor(propertyName, rawValue, indexPattern) {
+    super(propertyName, rawValue);
+    this._indexPattern = indexPattern;
+  }
+
+  getHtmlDisplayValue() {
+
+    if (typeof this._rawValue === 'undefined') {
+      return '-';
+    }
+
+    const field = this._indexPattern.fields.byName[this._propertyName];
+    if (!field) {
+      return '-';
+    }
+    const htmlConverter = field.format.getConverterFor('html');
+    return  (htmlConverter) ? htmlConverter(this._rawValue) :
+      field.format.convert(this._rawValue);
+  }
+
+  isFilterable() {
+    const field = this._indexPattern.fields.byName[this._propertyName];
+    return field && field.type === 'string';
```

Comment:
Why are you limiting to just string fields? Saved search table allows for creating filters on numerical fields, ip addresses, time fields, and more

<img width="300" alt="Screen Shot 2019-04-02 at 11 15 02 AM" src="https://user-images.githubusercontent.com/373691/55422351-b08beb00-5538-11e9-94f1-4657b93a019d.png">

Reply:
you're right, this limitation no longer makes sense. it's a carry-over from region-maps, but that only filters on join-fields, which are strings.

I would _not_ filter on numerical values. The majority of them are ordinal values, and not sure how useful that is to filter on exact measurements. (e.g. filtering on 7682 bytes seems not a natural way of using Discover).

Reply:
discussed in person. adding numbers as well.